File: Day49/SingleNo.java
import java.util.*;

class SingleNo {
    public int singleNumber(int[] nums) {
        int result = 0;
        for (int num : nums) {
            result ^= num;
        }
        return result;
    }
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        // Read array length
        int n = scanner.nextInt();
        int[] nums = new int[n];
        for (int i = 0; i < n; i++) {
            nums[i] = scanner.nextInt();
        }
        SingleNo sol = new SingleNo();
        System.out.println(sol.singleNumber(nums));
    }
}

File: Day49/SingleNo.py
class Solution(object):
    def singleNumber(self, nums):
        result = 0
        for num in nums:
            result ^= num
        return result

if __name__ == "__main__":
    n = int(input())
    nums = list(map(int, input().split()))
    solution = Solution()
    print(solution.singleNumber(nums))
